File: app/src/main/java/com/cliniva/enventory/ui/payment/PaymentFragment.java
package com.cliniva.enventory.ui.payment;


import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.cliniva.enventory.R;
import com.cliniva.enventory.ui.base.BaseFragment;

/**
 * A simple {@link Fragment} subclass.
 */
public class PaymentFragment extends BaseFragment {


    public PaymentFragment() {
        // Required empty public constructor
    }

    @Override
    public int getLayoutRes() {
        return R.layout.fragment_payment;
    }

    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
    }
}

File: app/src/main/java/com/amirsons/inventory/ui/fragment/payment/PaymentFragment.kt
package com.amirsons.inventory.ui.fragment.payment


import android.os.Bundle
import android.view.View

import com.amirsons.inventory.R
import com.amirsons.inventory.ui.base.BaseFragment

/**
 * A simple [BaseFragment] subclass.
 */
class PaymentFragment : BaseFragment(), PaymentView {

    private lateinit var presenter: PaymentPresenter

    override val contentLayout: Int
        get() = R.layout.fragment_payment

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        presenter = PaymentMvp(this)
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
    }
}
